src/plot.py

```diff
@@ -0,0 +1,147 @@
+import pandas as pd
+from great_tables import GT, html, md, px, style
+from great_tables import loc as gt_loc
+
+
+def ipc_table(df, title):
+    cur = pd.Timestamp.now()
+    return (
+        GT(df, rowname_col="phase")
+        .fmt_number(columns="population", decimals=0, use_seps=True)
+        .fmt_number(columns="pt_change", decimals=0)
+        .fmt_percent(columns="proportion", decimals=0)
+        .data_color(columns=["population"], palette="Reds", domain=[0, 6000000])
+        .data_color(columns=["proportion"], palette="Reds", domain=[0, 1])
+        .data_color(
+            columns=["pt_change"],
+            palette=["#2ecc71", "#a6d96a", "#ffffff", "#f46d43", "#d73027"],
+            domain=[-25, 25],
+        )
+        .cols_label(
+            population=html("Population in phase"),
+            proportion=html("Proportion of analyzed population"),
+            pt_change=html("Percent point change *"),
+        )
+        .tab_header(
+            title=md(title),
+        )
+        .tab_source_note(
+            source_note=html(
+                f"""
+                Updated as of {cur.strftime("%-d %b %Y")}. Referencing the latest data
+                that overlaps with the current date. See further details from the
+                <a href='https://www.ipcinfo.org/ipc-country-analysis/en/'>IPC website
+                </a>.<br><br> * A value of 0 may also indicate that no comparison
```

Comment:
a value of 0 in what column. In table on deployed app sometimes there are 0's sometimes just grey fill. Maybe need a note about grey fill?